config/routes.rb

```diff
@@ -13,6 +13,8 @@
      get 'covid-19', to: 'covid#index'
      get 'recent', to: 'recent#index'
      get 'community', to: 'community#index'
+     get '/:id/share', to: 'email#new'
+     post '/:id/share', to: 'email#create'
```

Comment:
I think this may be the reason the other tests are failing. It seems like the new obituary form is currently routing to the mailer. Moving `resources :obituaries` above `namespace :obituaries` might solve the problem.